### ISSUE TITLE
Ducted fans still require next to no EC

### ISSUE THREAD

**Issue by sh1pman** [BobPalmer/FTT#166]
Out of all ducted fans, only the smallest one, Micro Ducted Fan Engine takes a proper amount of EC/s. Every other engine still requires a miniscule amount of power to run at full throttle. For example, the medium ducted fan takes just 1.02 EC/s.

**Comment by GHRyanK** [comment]
I agree that the ducted fans may need more re-balancing, but I'm more concerned that there's inconsistency in the fan EC consumption.  I actually think the micro-ducted fans are over-consuming relative to appropriate sized power sources - and they're consuming more than all the bigger fans.  20 EC/s puts it beyond the capacity of a reasonable number of small fuel cells and forces the use of larger fuel cell arrays or 0.625m reactors, which would be a big (maybe even unusable) hit on thrust-to-weight ratios.

High efficiency fuel cells (on the order of ~10kW per cubic foot) would likely be the real-world power supply for electric ducted fans.  While the PB-NUK has been my go-to power supply for small vehicles, I've done it with a bad taste in my mouth because the in-game RTG is severely overpowered compared to its real-world counterpart (probably <100W per cubic foot for a Cassini RTG).

**Comment by Lyneira** [comment]
In the versions I'm using (ExPack 0.8.2.0 and FTT 0.7.2.0) the ratio between thrust and EC consumption is exactly the same for all the ducted fans, so this was probably fixed a while ago.

As for the high EC consumption ratio, they seem intended for short periods of usage on batteries or with nuclear reactors, the FTT versions even state as much. I would suggest Near Future Electrical for more options regarding both. Or perhaps just mod up the conversion speed of the standard fuel cells, which does seem rather weak for this particular application.
